libcontainer/cgroups/fs/cpuset_v2.go

```diff
@@ -22,11 +22,14 @@ func (s *CpusetGroupV2) Name() string {
 }
 
 func (s *CpusetGroupV2) Apply(d *cgroupData) error {
+	if d.config.Resources.CpusetCpus == "" && d.config.Resources.CpusetMems == "" {
+		return nil
+	}
 	dir, err := d.path("cpuset")
```

Comment:
not directly related to this PR, but I'm not convinced of `d.path()` logic  in v2 `Apply()` functions.

We should check controller availability and maybe write `cgroup.subtree_control` as crun does?

Reply:
cc @giuseppe

Reply:
yes, I think we should try to enable controllers that are needed